src/es6-extensions.js

```diff
@@ -140,3 +140,47 @@ Promise.race = function (values) {
 Promise.prototype['catch'] = function (onRejected) {
   return this.then(null, onRejected);
 };
+
+function getAggregateError(errors){
+  if(typeof AggregateError === 'function'){
+    return new AggregateError(errors,'All promises were rejected');
+  }
+
+  var error = new Error('All promises were rejected');
+
+  error.name = 'AggregateError';
+  error.errors = errors;
```

Comment:
I wandering is it correct way of polyfilling this `AggregateError`

Reply:
I think this should be "good enough" as a polyfill.